Fixing Broken "Method" hyperlink in D3D12DescriptorHeap Page

codenameone-akshat opened the pull request:
Changing "Method" Hyperlink from docs.microsoft.com to https://docs.microsoft.com/en-us/windows/win32/api/d3d12/nn-d3d12-id3d12descriptorheap#methods

codenameone-akshat committed:
Fixing Broken "Method" hyperlink.

Changing "Method" Hyperlink from docs.microsoft.com to https://docs.microsoft.com/en-us/windows/win32/api/d3d12/nn-d3d12-id3d12descriptorheap#methods

PRMerger14 commented:
@codenameone-akshat : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Best to...

just lose that content, since it literally links to the line below it. Also remember that we mustn't put language codes in urls.
